scripts/gzip-all.js

```diff
@@ -0,0 +1,30 @@
+const path = require('path');
+const gzipAll = require('gzip-all');
+
+var walkSync = function(dir, filelist) {
```

Comment:
This function can be refactored to something much simpler I think
Something like
```js
const fs = require('fs');
function walkSync(path, filelist = []) {
  if (fs.statSync(path).isDirectory) 
    fs.readdirSync().forEach(v => walkSync(v, filelist));
  else filelist.push(path);
  return filelist;
}
```

Reply:
Done.